### DISCUSSION TITLE
Distinguish all entries that belong to the same request

### DISCUSSION THREAD

**Discussion by diaafares** [spatie/ray#86] [Ideas]
It will be nice if ray provides a way to distinguish all entries that belong to the same request or maybe grouping them?
or show you all the previous requests so you can navigate them and click on any request to show all things happened in that particular request like dumps, queries and mailables ...etc

**Comment by freekmurze** [comment]
What you could to is, somewhere early in your request (maybe in a service provider) put a line like this:

```php
ray()->newScreen('request ' . now()->timestamp);
```

This way you get a screen per request.
